registries/adapters/adapter.go

```diff
@@ -179,3 +182,14 @@ func getAPBRuntimeVersion(version string) (int, error) {
 	}
 	return runtime, nil
 }
+
+// manifest schema version
+func getSchemaVersion(response []byte) (int, error) {
+	mResp := manifestResponse{}
+	r := bytes.NewReader(response)
+	if err := json.NewDecoder(r).Decode(&mResp); err != nil {
+		log.Errorf("Error getting schemaVersion", err)
+		return 0, err
+	}
+	return mResp.SchemaVersion, nil
+}
```

Comment:
this feels weird we're parsing the response bytes multiple times from bytes to json. We do it in `getSchemaVersion`. And if it version 1 we do it again in `responseToSpec`. reading the same set of bytes parsing the json and loading the same response struct. Feels like once we've loaded it we should stick with it.